.github/workflows/build.yml

```diff
@@ -441,3 +441,42 @@ jobs:
             zetaclientd-ubuntu-20-amd64
 
 
+  discord_announcement:
+    #runs-on: ubuntu-latest
+    runs-on: ["zeta-runners"]
+    timeout-minutes: 15
+    if: startsWith(github.ref, 'refs/tags/')
+    needs:
+      - upgrade_path_testing
+  
+    steps:
+      - uses: actions/checkout@v3
+  
+      # - name: setup-git-credentials
+      #   uses: de-vri-es/setup-git-credentials@v2.0.8
+      #   with:
+      #     credentials: ${{ secrets.PAT_GITHUB_SERVICE_ACCT }}
+
+      - name: "SEND:DISCORD:MESSAGE"
+        if: ${{ github.event.inputs.DRY_RUN != 'true' }}
```

Comment:
This doesn't apply to this workflow. An updated if statement could be used to determine if it's a major version increase. If `Version = vX.X.0` it's a major version, anything else should be minor